e2e/tests/nightly/admin/user-admin-ui.spec.ts

```diff
@@ -0,0 +1,181 @@
+import UserAdminPage from 'app/page/admin-user-list-page';
+import { signInWithAccessToken } from 'utils/test-utils';
+import navigation, { NavLink } from 'app/component/navigation';
+import AdminTable from 'app/component/admin-table';
+import UserProfileInfo from 'app/page/admin-user-profile-info';
+
+describe('Admin', () => {
+  const userEmail = 'admin_test';
+
+  beforeEach(async () => {
+    await signInWithAccessToken(page);
+    await navigation.navMenu(page, NavLink.USER_ADMIN);
+  });
+
+  test('verify admin-table and admin-user-profile page reflect the same modules and active status', async () => {
```

Comment:
Hmm, this is a very long test. It would be better if it can be shortened or split into several smaller tests.

Reply:
ok! will do. Thanks!